src/Plugin/AbstractFX.hpp

```diff
@@ -55,7 +57,12 @@ class AbstractPluginFX : public Plugin
         std::memset(efxoutl, 0, sizeof(float)*bufferSize);
         std::memset(efxoutr, 0, sizeof(float)*bufferSize);
 
-        doReinit(true);
```

Comment:
Are we sure this line should be removed?

Reply:
I see in the other changes doReinit() is called in the implementation classes, so I guess this is fine. It might have been a little simpler to add a dummy doReinit() to Reverse.h, but everything checks out.